common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -478,11 +478,11 @@ open class MusicService : MediaBrowserServiceCompat() {
 
         override fun onPrepare(playWhenReady: Boolean) {
             val recentSong = storage.loadRecentSong()
```

Comment:
We could probably make this even easier by changing this line to: `val recentSong = storage.loadRecentSong() ?: return`, then we wouldn't need the null check at all.

Reply:
OK